monitor_packages/kubernetes/variables.tf

```diff
@@ -66,4 +66,9 @@ variable "group_notifications" {
   type        = bool
   description = "Whether or not to group notifications for individual items that meet the trigger condition. Defaults to true."
   default     = true
+}
+
+variable "kubernetes_data_source" {
+  type = string
+  description = "Sumo Logic Kubernetes data source. For eg: kubernetes_data_source=cluster=production"
```

Comment:
Should it be more like cluster=production?

Reply:
It will be kubernetes_data_source="cluster=production", updated the description.